Sources/TestsCommon/MongoError+Equatable.swift

```diff
@@ -1,66 +1,60 @@
 import Foundation
 import MongoSwift
 
-extension RuntimeError: Equatable {
-    public static func == (lhs: RuntimeError, rhs: RuntimeError) -> Bool {
-        switch (lhs, rhs) {
-        case (.internalError(message: _), .internalError(message: _)),
-             (.authenticationError(message: _), .authenticationError(message: _)),
-             (.compatibilityError(message: _), .compatibilityError(message: _)):
-            return true
-        case let (
-            .connectionError(message: _, errorLabels: lhsLabels),
-            .connectionError(message: _, errorLabels: rhsLabels)
-        ):
-            return sortAndCompareOptionalArrays(lhs: lhsLabels, rhs: rhsLabels, cmp: { $0 < $1 })
-        default:
-            return false
-        }
+extension InternalError: Equatable {
+    public static func == (_: InternalError, _: InternalError) -> Bool {
+        return true
     }
 }
 
-extension ServerError: Equatable {
-    public static func == (lhs: ServerError, rhs: ServerError) -> Bool {
-        switch (lhs, rhs) {
-        case let (
-            .commandError(code: lhsCode, codeName: _, message: _, errorLabels: lhsErrorLabels),
-            .commandError(code: rhsCode, codeName: _, message: _, errorLabels: rhsErrorLabels)
-        ):
-            return lhsCode == rhsCode
-                && sortAndCompareOptionalArrays(lhs: lhsErrorLabels, rhs: rhsErrorLabels, cmp: { $0 < $1 })
-        case let (
-            .writeError(writeError: lhsWriteError, writeConcernError: lhsWCError, errorLabels: lhsErrorLabels),
-            .writeError(writeError: rhsWriteError, writeConcernError: rhsWCError, errorLabels: rhsErrorLabels)
-        ):
-            return lhsWriteError == rhsWriteError
-                && lhsWCError == rhsWCError
-                && sortAndCompareOptionalArrays(lhs: lhsErrorLabels, rhs: rhsErrorLabels, cmp: { $0 < $1 })
-        case let (
-            .bulkWriteError(
-                writeErrors: lhsWriteErrors,
-                writeConcernError: lhsWCError,
-                otherError: lhsOther,
-                result: lhsResult,
-                errorLabels: lhsErrorLabels
-            ),
-            .bulkWriteError(
-                writeErrors: rhsWriteErrors,
-                writeConcernError: rhsWCError,
-                otherError: rhsOther,
-                result: rhsResult,
-                errorLabels: rhsErrorLabels
-            )
-        ):
-            let cmp = { (l: BulkWriteError, r: BulkWriteError) in l.index < r.index }
-
-            return sortAndCompareOptionalArrays(lhs: lhsWriteErrors, rhs: rhsWriteErrors, cmp: cmp)
-                && lhsWCError == rhsWCError
-                && sortAndCompareOptionalArrays(lhs: lhsErrorLabels, rhs: rhsErrorLabels, cmp: { $0 < $1 })
-                && lhsResult == rhsResult
-                && lhsOther?.localizedDescription == rhsOther?.localizedDescription
-        default:
-            return false
+extension AuthenticationError: Equatable {
```

Comment:
do you recall why we added these equatable conformances in the first place rather than just using the matchers defined here? https://github.com/Quick/Nimble#swift-error-handling